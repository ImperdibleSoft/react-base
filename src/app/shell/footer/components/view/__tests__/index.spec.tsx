import * as React from 'react';
import { shallow } from 'enzyme';

import Footer from '..';

describe('Footer', () => {
  it('should render a valid footer', () => {
    const component = shallow(<Footer appVersion="1" fixedSidebar={false} />);

    expect(component).toMatchSnapshot();
  });
});
