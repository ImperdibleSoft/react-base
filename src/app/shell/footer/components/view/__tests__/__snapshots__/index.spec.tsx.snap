// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render a valid footer 1`] = `
<Wrapper
  fixedSidebar={false}
>
  <Content
    maxWidth={1280}
  >
    <Text>
      React Base
       
      v1
       © is a registered property of 
      ImperdibleSoft
       ©
       
      2019
    </Text>
  </Content>
</Wrapper>
`;
